how-to/integrate-with-excel/client/src/bootstrapper.ts

```diff
@@ -0,0 +1,24 @@
+import { fin } from "@openfin/core";
```

Comment:
I think we still need to register a platform which is why you are getting the broken icon in the menu for the browser.

Reply:
Done